### DISCUSSION TITLE
Add disconnect timeout configuration to the Server Settings menu

### DISCUSSION THREAD

**Discussion by gonenoculer5** [FakeFishGames/Barotrauma#14581] [Ideas and suggestions]
Hello!

I, a long time player of Barotrauma have always thought it a game better played with friends. But, unfortunately some of my friends have lower end PC's; and when playing with mods like More Level Content, Barotraumatic, etc, that can mean their lower end devices take longer to load in levels and complete equality checks. Unfortunately this means they often can't play with my group, and this is something I would like to remedy.

I would like to propose the idea to add a configuration option for the duration of the level loading timeout, with the default setting being what it currently is. This would allow players with lower end hardware to enjoy the game they can otherwise play fine. 

Thanks!

**Comment by lIllIIlI** [comment]
This is an issue I see in every campaign running Dynamic Europa and a huge mod list. First few biomes are fine, but 3rd onwards, when we have higher tier subs, more items, more players, etc. Loading takes a lot longer and basically everyone gets kicked out by the server and has to rejoin.

**Comment by Ek-Videogames** [comment]
Even in vanilla, syncing weirdness or dedicated server performance can cause players to timeout while waiting for the server to send a "response" if the server takes a second or two too long to load the round for any reason.
